src/wasm/wasm-type.cpp

```diff
@@ -1502,6 +1518,9 @@ bool TypeBounder::lub(const Rtt& a, const Rtt& b, Rtt& out) {
 
 template<typename T, typename F>
 std::ostream& TypePrinter::printChild(T curr, F printer) {
+  if (exceededLimit()) {
+    return os << "..!";
```

Comment:
How is this going to be printed? `exceededLimit` also prints `?`, so..

Reply:
Oh yeah, we don't need both prints. I like this one better FWIW. Otherwise LGTM.

Reply:
Oh, good point... fixed.